src/DefaultIcon.js

```diff
@@ -18,7 +18,10 @@ let iconMap = {
   PreviousPage: () => <span>←</span>,
   Previous5Pages: () => <span>⇜</span>,
   Next5Pages: () => <span>⇝</span>,
+  Refresh: ({ onClick }) => <span onClick={onClick}>⟳</span>,
```

Comment:
In other places, we’ve just wrapped the icon with whatever which handles the onClick (e.g. using IconButton in grey vest or even just a div). We should probably stay consistent with that pattern

Reply:
done